lecture_3/03_project/calculator/internal/engine/calculatorEngine.go

```diff
@@ -0,0 +1,62 @@
+package engine
+
+import (
+	"context"
+	"log"
+)
+
+type Calculator struct {
```

Comment:
Struct and file should be named identically. Either rename the struct or the file (I'm fine with both, but file is easier).